beacon-chain/rpc/validator_server.go

```diff
@@ -28,36 +28,36 @@ type ValidatorServer struct {
 // beacon state, if not, then it creates a stream which listens for canonical states which contain
 // the validator with the public key as an active validator record.
 func (vs *ValidatorServer) WaitForActivation(req *pb.ValidatorActivationRequest, stream pb.ValidatorService_WaitForActivationServer) error {
-	if vs.beaconDB.HasValidator(req.Pubkey) {
+	if vs.beaconDB.HasValidators(req.PublicKey) {
```

Comment:
`req.PublickKeys`

Reply:
if one validator doesnt exist, this whole method fails ?

Reply:
HasValidators returns true if one validator exists

Reply:
that doesnt sound right, if out of 10 only one existing would be a failure. it should only return true if all the validator exist. If some validators do not exist you could return false and those validators pubkey.

Reply:
made it to return true only if all the validators exists